app/src/main/java/au/com/shiftyjelly/pocketcasts/ui/MainActivity.kt

```diff
@@ -994,27 +987,19 @@ class MainActivity :
 
     override fun onMiniPlayerHidden() {
         val padding = binding.bottomNavigation.height
-        updateSnackbarPosition(padding)
+        updateSnackbarPosition(0)
         settings.updateBottomInset(0)
     }
 
     private fun updateSnackbarPosition(bottomPadding: Int) {
         binding.snackbarFragment.updateLayoutParams<ViewGroup.MarginLayoutParams> {
-            bottomMargin = bottomPadding
-        }
-    }
-
-    private fun setupSnackbarPosition() {
-        // we manually position the snackbar to account for the mini player and bottom navigation, consume the insets so extra padding isn't added to the snackbar
-        ViewCompat.setOnApplyWindowInsetsListener(binding.snackbarFragment) { _, _ ->
-            WindowInsetsCompat.CONSUMED
```

Comment:
This caused me an issue for a while. I learned it isn't good to consume the insets as nothing else can use them. The filters create page didn't get any insets until I removed this.